doc/howToBuildADuck.md

```diff
@@ -103,6 +103,14 @@ Download USB to UART Bridge VCP Driver from [here](https://www.silabs.com/produc
 * Make sure you click on the folder that says Legacy MacVCP Driver and then click on ‘Silicon Labs VCP Driver.pkg’
 * Once it is finished installing, go to Mac System Preferences -> Security and Privacy -> General. Make sure Silicon Labs is allowed.
 
+#### Ubuntu Linux Platforms: 
+*  The driver is already installed on your machine due to latest kernels. you can check to see by running the following commands. 
+```ls -al /lib/modules/"$(uname -r)"/kernel/drivers/usb/serial/usbserial.ko
+```
+```
+ls -al /lib/modules/"$(uname -r)"/kernel/drivers/usb/serial/cp210x.ko
+```
```

Comment:
This is not only valid for Ubuntu. It should be valid for all Linux distributions. And I think a `modinfo` (installed modules) or a `lsmod` (loaded modules) is better suited to determine if a module is installed than a normal `ls`.